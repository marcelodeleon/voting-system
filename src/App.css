.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
.modal-container{
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background-color: rgba(255, 255, 255, 0.75)
}
.modal {
  font-size: 20px;
  color: rgb(0, 28, 104);
  font-family: Arial Black;
  position: absolute;
  top: 130px;
  left: 190px;
  right: 190px;
  bottom: 130px;
  border: 5px solid rgba(0, 19, 48, 0.74);
  background: #fff;
  overflow: auto;
  text-align: center;
  background-color: rgba(132, 171, 255, 0.562)
}
.my-form {
  text-align:auto;
  font-size: 20px;
  font-family: Arial Black;
}
