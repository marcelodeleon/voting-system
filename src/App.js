import React from 'react';
import './App.css';
import { BrowserRouter, Switch, Route } from 'react-router-dom';
import UserRegistration from './components/users/UserRegistration';
import ResultsPage from './components/users/ResultsPage';
import Vote from './components/users/Vote';
import Election from './components/admin/Election';
import Home from './components/Home';
import VerifyEmail from './components/users/VerifyEmail';
import ResetPassword from './components/users/ResetPassword';
import NewPassword from './components/users/NewPassword';

function App() {
  return (
    <BrowserRouter>
      <Switch>
        <Route exact path="/" component={Home} />
        <Route exact path="/users/registration" component={UserRegistration} />
        <Route exact path="/users/result" component={ResultsPage} />
        <Route exact path="/vote" component={Vote} />
        <Route exact path="/admin/election" component={Election} />
        <Route exact path="/verify" component={VerifyEmail} />
        <Route exact path="/reset" component={ResetPassword} />
        <Route exact path="/newpass" component={NewPassword} />
      </Switch>
    </BrowserRouter>
  );
}

export default App;
