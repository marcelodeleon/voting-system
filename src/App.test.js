import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom'
import 'mutationobserver-shim';
import App from './App';

global.MutationObserver = window.MutationObserver;

test('Renders App correctly', () => {
  const { container } = render(<App />, { wrapper: MemoryRouter });

  expect(container.firstChild).toMatchSnapshot()
});
