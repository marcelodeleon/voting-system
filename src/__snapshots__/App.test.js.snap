// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders App correctly 1`] = `
<div>
  <form
    class="sc-jrAGrp wPOmF"
  >
    <h1>
      Login
    </h1>
    <div
      class="sc-kEjbxe jtAWeG"
    >
      <label>
        Email:
        <input
          name="email"
          style="width: 30rem;"
          type="email"
          value=""
        />
      </label>
      <label>
        Password:
        <input
          name="password"
          style="width: 30rem;"
          type="password"
          value=""
        />
      </label>
      <input
        class="sc-iqHYGH knFwIE"
        type="submit"
        value="Log In"
      />
      <a
        href="/users/registration"
      >
        ¿No tienes cuenta? Click aquí para registrar un usuario
      </a>
    </div>
  </form>
</div>
`;
